.github/workflows/build.yml

```
@@ -73,6 +79,27 @@ jobs:
+      RELEASE_VERSION: 0.0.0-ci.${{ github.run_number }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v6
+
+      - name: Setup Rust
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Inject workspace release version
+        run: python .github/scripts/set-rust-workspace-version.py --version "$RELEASE_VERSION"
+
+      - name: Dry run: publish core
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)